src/VAC/VectorAnimationComplex/VAC.h

```diff
@@ -477,6 +477,9 @@ public slots:
     // Transform tool
     TransformTool transformTool_;
     friend class TransformTool;
+
+    double pasteDeltaX_;
+    double pasteDeltaY_;
```

Comment:
I feel safer if these are zero-initialized when declared. :)

Reply:
I added initializations in constructors